cmd/server/cadence/server.go

```diff
@@ -70,9 +70,6 @@ func newServer(service string, cfg *config.Config) common.Daemon {
 
 // Start starts the server
 func (s *server) Start() {
-	if _, ok := s.cfg.Services[s.name]; !ok {
```

Comment:
No need to check here. This is checked down the line in `server.startService` when config is actually needed.